oscarapi/views/basket.py

```diff
@@ -271,6 +271,6 @@ def get_queryset(self):
         basket = get_object_or_404(operations.editable_baskets(), pk=basket_pk)
         prepped_basket = operations.prepare_basket(basket, self.request)
         if operations.request_contains_basket(self.request, prepped_basket):
-            return prepped_basket.lines
+            return prepped_basket.all_lines()
```

Comment:
ALTIJD (en wordt nu al overal in oscarapi gebruikt behalve hier). 

Kijk maar: https://github.com/django-oscar/django-oscar/blob/master/src/oscar/apps/basket/abstract_models.py#L115